starter-code/src/functions-and-arrays.js

```diff
@@ -1,6 +1,35 @@
 // Find the maximum
+function maxOfTwoNumbers(a,b) {
+  if(a > b){
+    //console.log(a + "is the larger number.");
+    return a;
+  }else if(a < b){
+    //console.log( b + "is the largest number")
+    return b;
+  }else{
+    //console.log("Numbers are equal.")
+  } return a,b; 
+}
+
+
+//Finding Longest Word
+function findLongestWord(array) {
+  var longestWordLength = 0;
+  var longestWord;
+
+  if (array.length > 0) {
+    for (let i = 0; i < array.length; i++) {
+      if (array[i].length > longestWordLength){
+        longestWordLength = array[i].length;
+        longestWord = array[i];
+      }    
+    } return longestWord;
+  }
+  else
```

Comment:
This `else return` is not need it since javascript will return `undefined` as default